src/installer/corehost/cli/corehost_context_contract.h

```diff
@@ -25,7 +25,8 @@ enum class coreclr_delegate_type
     winrt_activation,
     com_register,
     com_unregister,
-    load_assembly_and_get_function_pointer
+    load_assembly_and_get_function_pointer,
+    unknown, // Sentinel value for determining the last known delegate type
```

Comment:
I would call it `__last` or something like that - make it stand out as "internal" value (double underscores) and "unknown" doesn't sound right to me, but I guess `__unknown` would be OK.

Reply:
I see now that `__last` is the convention in this area of the code so I'll go with that.